### PR TITLE
tasklets: allow to dispatch raw functions

### PR DESCRIPTION
**Pull request by andlaus**

### CODE REVIEW

**Comment by andlaus** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks. I will merge it tomorrow if @atgeirr or  @dr-robertk don't object.

**Comment by atgeirr** [comment]
Please wait, it is on my list, and I'll get to it soon!

**Comment by andlaus** [comment]
@atgeirr: I'd like to merge this ASAP: this does not currently affect any simulators and it makes it easier to dispatch asynchronous jobs, i.e., it is a worthwhile improvement with zero potential for regressions.

**Comment by atgeirr** [comment]
> @atgeirr: I'd like to merge this ASAP: this does not currently affect any simulators and it makes it easier to dispatch asynchronous jobs, i.e., it is a worthwhile improvement with zero potential for regressions.

Will compile and test after I finished testing the std::mutex stuff. So I'll likely merge it later this afternoon, if no issues are encountered.

File: ewoms/parallel/tasklets.hh [hunk 1]
```diff
@@ -61,6 +61,26 @@ private:
...
+/*!
+ * \brief A simple tasklet that runs a function that returns void and does not take any
+ *        arguments a given number of times.
+ */
+template <class Fn>
+class FunctionRunnerTasklet : public TaskletInterface
+{
+public:
+    FunctionRunnerTasklet(const FunctionRunnerTasklet&) = default;
+    FunctionRunnerTasklet(int numInvokations, const Fn& fn)
```
Comment (atgeirr) [inline ewoms/parallel/tasklets.hh]: Should be `numInvocations` (with a c), also in other places.

**Comment by atgeirr** [comment]
Looks fine apart from typo, can be merged when fixed and green.

**Comment by andlaus** [comment]
jenkins build this please

**Pull request merged**